05_syntax_lab/01.py

```diff
@@ -1,5 +1,15 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Syntax excersize #1
+"""
+bigNum = 0

```

Comment:
Variable names in python are usually written lowercase only, so `bignum` is a better name. For long names with multiple words you can use underscores for example: `the_read_huge_number`